website/static/js/profile.js

```diff
@@ -580,6 +582,15 @@ var ListViewModel = function(ContentModel, urls, modes) {
         return self.contents().length > 1;
     });
 
+    self.institutionObjectsEmpty = ko.pureComputed(function(){
+        for (var i=0; i<self.contents().length; i++) {
```

Comment:
As I understand this loop, the function will return `true` if the first item in the list is empty. I believe the `false` return should be inside the conditional, not outside.